CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,65 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Validate if Deferred Account is set for Financial Type
+   * when Deferred Revenue is enabled
+   *
+   * @return string
+   *
+   */
+  public static function validateTogglingDeferredRevenue() {
+    $deferredFinancialType = self::getDeferredFinancialType();
+    $message = ' Please revise the error message to the following:
```

Comment:
Please remove 'Please revise the error message to the following:' from the error message itself!